rules/windows/process_creation/proc_creation_win_wmic_recon_volume.yml

```diff
@@ -0,0 +1,35 @@
+title: System Disk And Volume Reconnaissance Via Wmic.EXE
+id: c79da740-5030-45ec-a2e0-479e824a562c
+related:
+    - id: d85ecdd7-b855-4e6e-af59-d9c78b5b861e
+      type: similar
+status: experimental
+description: |
+    An adversary might use WMI to discover information about the system, such as such as the volume name, size,
+    free space, and other disk information. This can be done using the `wmic` command-line utility, and has been
+    observed being used by threat actors such as Volt Typhoon.
```

Comment:
```suggestion
    An adversary might use WMI to discover information about the system, such as the volume name, size,
    free space, and other disk information. This can be done using the `wmic` command-line utility and has been
    observed being used by threat actors such as Volt Typhoon.
```